k8s/modules-cid-ckecker.tf

```diff
@@ -0,0 +1,29 @@
+module "marketdeals_calibration" {
+  count           = local.is_dev_envs
+  source          = "../modules/cid_checker"
+  bucket_name     = "calibration"
+  force_destroy   = true
+  tags            = module.generator.common_tags
+  oidc            = local.oidc_URL
```

Comment:
I would suggest adding the getting `oidc_URL` logic directly in the module. 
Like just to add some that we have in the k8s folder.
It should increase a common understanding of the code configuration.

Reply:
Copied oidc_URL to locals.tf in cid-checker module.